templates/docs/examples/patterns/tiered-list/50-50-with-description.html

```diff
@@ -6,7 +6,7 @@
 {% block standalone_css %}patterns_all{% endblock %}
 
 {% block content %}
-{%- call(slot) tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
+{%- call(slot) tiered_list(variant=3) -%}
```

Comment:
While "simpler" it is completely obscured. What does the "3" mean? How would anyone know what it means and what value to choose? Would designers and developers understand them in the same way? What if we need to add another slightly different version of existing one? Throwing in just another number (like `variant=5`) if that's a new modification for responsiveness of `variant=1` seems weird and confusing.

With `is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false` I at least know more or less what options I'm setting. With `variant=3` I know nothing.

Developers are likely used to using words for naming Vanilla options, because they have to do it in CSS using class names. This way, the name of a variant is at least somewhat descriptive. This also allows us to have separate flags if there are independent options to change, rather than hiding everything behind a number.

I guess I wouldn't be so opposed to that if instead of numbers we would use some more descriptive names, like "full-width" "full-width-on-large" "full-width-on-medium" … but, with macros we can't really enforce the values with any enum, so it becomes a free string (which is quite error prone). The named parameters at least give us a bit of control over what is being passed to the macro.

Overall, I wonder why do we want to change this? Has current approach turned out to be so complex people don't know how to use it? Is the proposed one actually better in this regard?